build.sbt

```diff
@@ -17,7 +14,7 @@ lazy val `scalajs-bundler-linker` =
 
 val `sbt-scalajs-bundler` =
   project.in(file("sbt-scalajs-bundler"))
-    .enablePlugins(BuildInfoPlugin)
+    .enablePlugins(SbtPlugin, BuildInfoPlugin)
```

Comment:
What is the difference with setting `sbtPlugin := true`?

Reply:
It also enables `scripted`. In sbt 1.x you have to use that to use `scripted` tests.